src/ergtable.h

```diff
@@ -0,0 +1,140 @@
+#ifndef ERGTABLE_H
+#define ERGTABLE_H
+
```

Comment:
Much as I love C#, I'm wondering why you have the implementation in the header. 

You probable already know, but if you add ergtable.cpp to the project, QtCreator in Windows will put the implementations in the cpp file for you from the context menu activated on the class name (ergtable):


<img width="702" alt="image" src="https://github.com/cagnulein/qdomyos-zwift/assets/29954900/923d056a-53ca-4afd-8bc2-7a77c9a8014c">

Reply:
Not a fan too, I did because it was quicker to write. I'm usually write code from the phone with my daughter on my legs :)